blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,98 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is an `enterprise`_ feature.
```

Comment:
wouldn't it be better to word it like "This plugin is part of the `enterprise`_ edtition."?